src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -194,7 +193,11 @@ public void testUpdateFeedbackResponseComment() throws Exception {
         ______TS("fail: invalid params");
         
         frComment.courseId = "invalid course name";
-        verifyExceptionThrownWhenUpdateFrComment(frComment, "not acceptable to TEAMMATES as a/an course ID");
+        String expectedError = getPopulatedErrorMessage(
```

Comment:
Let's go back to the string. It's fine if the frontend use the constant. The test should clearly show the behavior 'as seen by user', not 'as generated by the software'. This duplication allows cross checking of all places where a constant is used, instead of checking the single place where it is defined.